src/toil/utils/toilLaunchCluster.py

```diff
@@ -108,7 +109,7 @@ def main() -> None:
     options = parser.parse_args()
     set_logging_from_options(options)
 
-    tags = create_tags_dict(options.tags) if options.tags else dict()
+    tags = create_tags_dict(options.tags) if options.tags else build_tag_dict()
```

Comment:
These functions could have better names to explain why we need two of them that kind of do the same thing. Or we could unify them somehow.